.eslintrc

```diff
@@ -60,7 +60,6 @@ rules:
   indent: [2, 2, {SwitchCase: 1}]
   key-spacing: [2, {mode: "minimum"}]
   keyword-spacing: 2
-  linebreak-style: [2, "unix"]
```

Comment:
Rather than removing this rule, can you change its value to the default value? The net result would be the same as removing it, but having the explicit value might prevent people in the future from readding the "wrong" rule (at least they might be prompted to look at the commit history for that line).